all-the-icons.el

```diff
@@ -188,8 +188,8 @@
     ("pro"          all-the-icons-alltheicon "prolog"         :height 1.1  :face all-the-icons-lmaroon)
     ("proog"        all-the-icons-alltheicon "prolog"         :height 1.1  :face all-the-icons-lmaroon)
     ("py"           all-the-icons-alltheicon "python"         :height 1.0  :face all-the-icons-dblue)
+    ("idr"          all-the-icons-fileicon "idris"            :face all-the-icons-red)
```

Comment:
Shouldn't we set the height property here to `1.0`?

Reply:
Honestly I don't know, this worked for me.